### PR TITLE
Fix dataview bbox bug when no query rewrite data exists

### LINKED ISSUE #457: Dataviews without query rewrite data and with a bbox fail

### PR DESCRIPTION
**Pull request by jgoizueta**
Fixes #457

### CODE REVIEW

**Comment by jgoizueta** [comment]
@rochoa can you CR this?

File: test/acceptance/dataviews/overviews.js [hunk 1]
```diff
@@ -218,16 +250,32 @@ describe('dataviews using tables with overviews', function() {
...
+            }
+            assert.deepEqual(formula_result, {"operation":"sum","result":15,"nulls":0,"type":"formula"});
+
+            testClient.drain(done);
+        });
+    });
+
     describe('filters', function() {
 
         describe('category', function () {
```
Comment (rochoa) [inline test/acceptance/dataviews/overviews.js]: Why so many changes here?

Comment (jgoizueta) [inline test/acceptance/dataviews/overviews.js]: The test code was messy (in the layout) and fixed it. I also added test to check dataviews with overviews also don't fail when the bbox parameter is present (this was not failing)

Comment (jgoizueta) [inline test/acceptance/dataviews/overviews.js]: The one test that detects the problem is in this commit: https://github.com/CartoDB/Windshaft-cartodb/pull/458/commits/5989ab344d450155f71885e8e32d6a6b59b156ac

The rest is just fixing layout/having more coverage for similar cases

**Pull request merged**